model/roles.go

```diff
@@ -273,6 +287,24 @@ func LoadRoleManifest(manifestFilePath string, releases []*Release) (*RoleManife
 		rolesManifest.rolesByName[role.Name] = role
 	}
 
+	// NOTE: Consider merging the error lists into a larger one.
```

Comment:
Done with commit 33e85568c4acea62386712b69809c319904f845e
(Output on HCF manifest to be added to description soon)